### DISCUSSION TITLE
so-allow with Wazuh agent and IPv6

### DISCUSSION THREAD

**Discussion by rdemars-zcorum** [Security-Onion-Solutions/securityonion#3373] [Unsupported Versions]
On the Security Onion manager when trying to allow a Wazuh agent with an IPv6 address using the following command:
/usr/sbin/so-allow -w -i fc80::998
nothing seems to happen with adding the IPv6 address.
Anyone have any ideas?

**Comment by TOoSmOotH** [comment]
Although we support the sniffing of IPv6 on our monitor interface, we do not support IPv6 on out management interface. You would need to use IPv4 to on all of your clients.
